src/level_four/factory_reset/cyt_factory_reset.c

```diff
@@ -68,7 +65,8 @@ void cyt_factory_reset() {
             break;
 
         case FACTORY_RESET_CONFIRM:
-            confirm_scr_init(ui_text_confirm_factory_reset);
+            ui_scrollable_page(ui_heading_confirm_action, ui_text_confirm_factory_reset,
```

Comment:
In some similar cases, instead of Confirm action heading, we could give the heading of that setting, i.e. `factory reset` in this case.

Reply:
I think this feedback should come from design team. 

We can fix text fitment issues in this PR. Text changes can be taken up later